checkers_bot_tournament/bots/greedycat.py

```diff
@@ -5,25 +5,27 @@
 
 import copy
 
+
 class GreedyCat(Bot):
+    """
+    Maximise the length of my move_list after my opponent's best move
```

Comment:
No this one is "Maximise my material after 3 layers of search"... whoops some of the comments are a bit outdated

Reply:
oh oops feel free to change it on my branch or next pr idm